google/blockly
# chore: clean up some underscores

## The basics


## The details
### Resolves

General code cleanup.

### Proposed Changes

Stop using underscores for private fields and methods in `block_drag_surface.ts` and `bubble.ts`. Where possible, remove casts and `!` markers.

#### Behavior Before Change

No change.

#### Behavior After Change

No change.

### Reason for Changes

General code cleanup: standardize on non-underscored private variable names, and avoid casts so the compiler can better tell us when we make a breaking change.

### Test Coverage

Mocha tests, and manual testing:
- Open up the playground and get a block with a mutator.
- Drag the mutator around by various parts of the bubble (check for no change in bubble behaviour)
- Add multiple blocks and drag them around the workspace, singly and connected (check for no changes in block drag surface).

### Documentation
None.

### Additional Information

This change was made possible by knowing which properties are private and which are public!
I did not touch anything that's protected.
